src/kerberos/client/generators.rs

```diff
@@ -68,9 +68,9 @@ const MD5_CHECKSUM_TYPE: [u8; 1] = [0x07];
 // https://www.rfc-editor.org/rfc/rfc4120#section-5.4.1
 pub const DEFAULT_AS_REQ_OPTIONS: [u8; 4] = [0x00, 0x81, 0x00, 0x10];
 
-// Renewable, Canonicalize, Enc-tkt-in-skey are on by default
+// Renewable, Canonicalize.
 // https://www.rfc-editor.org/rfc/rfc4120#section-5.4.1
-const DEFAULT_TGS_REQ_OPTIONS: [u8; 4] = [0x00, 0x81, 0x00, 0x08];
+const DEFAULT_TGS_REQ_OPTIONS: [u8; 4] = [0x00, 0x81, 0x00, 0x00];
```

Comment:
I found a mistake here after writing new tests. We don't need to set the `enc-tkt-in-skey` flag by default. It should be set only when the Kerberos U2U is negotiated